paf-mvp-frontend/src/main.ts

```diff
@@ -19,4 +24,17 @@ const showNotification = (type: NotificationEnum) => notificationService.showNot
 
 // TODO: avoid global declaration
 window.PAFUI ??= { promptConsent, showNotification };
-window.PAF ??= { getNewId, signPreferences, getIdsAndPreferences, refreshIdsAndPreferences, updateIdsAndPreferences };
+window.PAF = {
+  // If the queue has been defined outside of this script, will be replaced by it
```

Comment:
NIT: this comment is not relevant anymore

Reply:
Fixed